RUFAS/input_manager.py

```diff
@@ -289,7 +300,7 @@ def _validate_input_type_dynamic(self, variable_properties: Dict[str, Any], var_
         return validator(variable_properties, var_name, input_data_value)
 
     def _validate_csv_element(self, var_name: str, properties_blob_key: str, input_data: Dict[str, Any],
-                              eager_termination: bool) -> dict:
+                              eager_termination: bool, element_counter_and_validity: Dict[str, Any]) -> dict:
```

Comment:
Why is `element_counter_and_validity` Any? It's either bool or int. It seems the same problem exists in other functions as well.
```suggestion
                              eager_termination: bool, element_counter_and_validity: Dict[str, int|bool]) -> Dict[str, int|bool]:
```